src/map/enmity_container.cpp

```diff
@@ -309,12 +308,12 @@ void CEnmityContainer::SetCE(CBattleEntity* PEntity, uint16 amount)
     }
 }
 
-void CEnmityContainer::SetVE(CBattleEntity* PEntity, uint16 amount)
+void CEnmityContainer::SetVE(CBattleEntity* PEntity, const int32 amount)
```

Comment:
const value?

Reply:
It's const correct for mutators to take in constant values. These functions shouldn't be modifying the value that goes in, it's not intuitive behavior.

Reply:
they can't modify the value that goes in because it's passed by copy

Reply:
Not what I meant. If you call `SetThisThing(30)`, I expect ThisThing to equal 30 when it's done.